node/resolvers/directives/checkAdminAccess.ts

```diff
@@ -48,13 +58,15 @@ export class CheckAdminAccess extends SchemaDirectiveVisitor {
           hasApiToken,
           hasValidApiToken,
           hasStoreToken,
```

Comment:
Since the store token is not used in this directive, we should remove it for better code readability.

Reply:
we have it here just to have a metric to identify if this token is being provided incorrectly for this case.